logging/telemetryhook.go

```diff
@@ -87,7 +86,10 @@ func createAsyncHookLevels(wrappedHook logrus.Hook, channelDepth uint, maxQueueD
 					}
 					hook.Unlock()
 					if entry != nil {
-						hook.wrappedHook.Fire(entry)
+						err := hook.wrappedHook.Fire(entry)
```

Comment:
Do we also need to capture the error returned from Fire here (in func logTelemetry)?
logging/telemetry.go 228:	t.hook.Fire(entry)

Reply:
The Fire function on the `logging/telemetry.go 228` goes to `asyncTelemetryHook.Fire` which aways returns nil, so adding a logging there is correct, but won't really be helpful.